src/css/reset.scss

```diff
@@ -0,0 +1,117 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
Looks like we can get this from [npm](https://www.npmjs.com/package/reset-css), so we can avoid including this file.